feature/home/src/commonTest/kotlin/com/droidknights/app/feature/home/SponsorUiStateTest.kt

```diff
@@ -0,0 +1,66 @@
+package com.droidknights.app.feature.home
+
+import com.droidknights.app.core.model.Sponsor
+import com.droidknights.app.feature.home.model.SponsorsUiState
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.collections.shouldContainExactly
+import io.kotest.matchers.shouldBe
+
+class SponsorUiStateTest : BehaviorSpec({
+    given("다음과 같은 Sponsor가 주어졌을 때") {
```

Comment:
```suggestion
    given("다음과 같은 후원사 목록이 주어졌을 때") {
```

[테스트 코드가 명세서](https://tech.inflab.com/20230404-test-code/#%EC%84%B8%EB%B2%88%EC%A7%B8-%EC%9D%BD%EA%B8%B0-%EC%A2%8B%EC%9D%80-%ED%85%8C%EC%8A%A4%ED%8A%B8%EB%A5%BC-%EC%9E%91%EC%84%B1%ED%95%98%EB%9D%BC)라는 관점으로 생각하면 우리가 사용하는 언어로 작성하는 것도 좋을 것 같아요